### DISCUSSION TITLE
best change for playcover

### DISCUSSION THREAD

**Discussion by lemontea87** [PlayCover/PlayCover#540] [Ideas]
how to use keyboard and mouse at the same time to play Mobile Legend Bang Bang

**Comment by amirsaam** [comment]
This option is already in PlayCover as was before. You need to make a Keymapping for your needs.

**Reply by Zhusiyu123** [reply]
sorry how can I set that and I am in nervous.could you show me how?

**Comment by MEMORIEmusic** [comment]
> sorry how can I set that and I am in nervous.could you show me how?

It is written in the [docs](https://docs.playcover.io/keymapping/using_making_keymaps)
